cpp/src/arrow/engine/substrait/serde_test.cc

```diff
@@ -45,6 +52,15 @@ using internal::checked_cast;
 
 namespace engine {
 
+bool WriteParquetData(const std::string& path,
+                      const std::shared_ptr<fs::FileSystem> file_system,
+                      const std::shared_ptr<Table> input, const int64_t chunk_size = 3) {
```

Comment:
You only call this in one place and you never supply a custom chunk size.  Can you remove this parameter?  Or are you planning a test case that will manipulate it?  It doesn't seem relevant to the issue under test.